### ISSUE TITLE
Sticky Navbar or Footer

### ISSUE THREAD

**Issue by redoxahmii** [breadadams/scroller-motion#41]
The sticky elements do not seem to be working while using ScrollerMotion but if you remove that it starts working.
I was creating a scroll progress bar to sticky display on the bottom of the screen but with ScrollerMotion it does not seem to work whereas removing it makes it normal again and the positioning begins to work again.
Do tell if there is some workaround to this or if i am just not understanding something.

**Comment by redoxahmii** [comment]
You have to define layouts and then define sticky elements outside the ScrollerMotion wrapper to make it work fyi.

**Comment by breadadams** [comment]
> You have to define layouts and then define sticky elements outside the ScrollerMotion wrapper to make it work fyi.

@Redoxahmii yep you can move them outside the `ScrollerMotion` wrapper (with a Portal works too).

This has been asked [numerous times](https://github.com/breadadams/scroller-motion/issues/36#issuecomment-1445538596) so probably about time I add a note in the docs 😅.
